specification/ai/Azure.AI.Projects/agents/common/models.tsp

```diff
@@ -130,11 +133,12 @@ union AgentsApiResponseFormatMode {
  */
 model AgentsApiResponseFormat {
   /** Must be one of `text` or `json_object`. */
-  type?: ApiResponseFormat = ApiResponseFormat.text;
+  type?: ResponseFormat = ResponseFormat.text;
```

Comment:
was this naming change to align with swagger?

Reply:
Yes, In new swagger they have renamed this model.